docs/bigquery-usage.rst

```diff
@@ -283,20 +279,22 @@ Run a query which can be expected to complete within bounded time:
    >>> query = """\
    SELECT count(*) AS age_count FROM dataset_name.person_ages
    """
-   >>> results = client.run_sync_query(query, timeout_ms=1000)
+   >>> job = client.run_sync_query(query)
+   >>> job.timeout_ms = 1000
```

Comment:
Is this not a desirable feature to have in the constructor? ISTM an accidental consequences of our `getter`/`setter` design on some of the more complex API nouns.